engine_timeseries.ipynb

```diff
@@ -294,21 +343,23 @@
     "    df,\n",
     "    time_feature_name=time_feature_name,\n",
     "    id_feature_name=id_feature_name\n",
-    ")"
+    ")\n",
+    "\n",
+    "features"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Step 3: Build Trainee and make a time series prediction\n",
+    "## Step 3: Train a Trainee and Analyze\n",
     "\n",
-    "The one change between regular analysis using non-time series data and time series data is that during the `train()` call, Engine calculates time information (deltas and rates) for each feature and adds this information in as additional features in your dataset. So, to utilize these new features which contain time information within the prediction, they have to be called as context features. The names of these features can be obtained after the `train()` call using the `get_cases()` call. Then, these can be passed in as context features to the `analyze()` and `react()` calls to make a time series prediction."
+    "The one change between regular analysis using non-time series data and time series data is that during the `train` call, Engine calculates time information (deltas and rates) for each feature and adds this information in as additional features in your dataset. So, to utilize these new features which contain time information within the prediction, they have to be called as context features. The names of these features can be obtained after the `train` call using the `get_cases` call. Then, these can be passed in as context features to the `analyze` and `react` calls to make a time series prediction."
```

Comment:
I think it would be useful to bold the comment about pulling in the deltas and rates as context features because this is an important, but non-obvious, piece of time series predictions.